SpatialGDK/Documentation/content/non-unreal-server-worker-types.md

```diff
@@ -13,33 +13,52 @@ In order to interact with each other, Unreal and non-Unreal server-worker instan
+* Instantiate the `ExternalSchemaInterface` class inside your game module. If you want to ensure that the SpatialOS worker connection registers your callbacks to receive initial network operations, you need to register the callbacks inside your game instance's `OnConnected` event callback.
+* To send SpatialOS component updates and commands, call the overloaded `SendComponentUpdate`, `SendCommandRequest`, or `SendCommandResponse` method for the relevant schema type.
+* To register callbacks for receiving SpatialOS component network operations, call the overloaded `OnAddComponent`, `OnComponentUpdate`, `OnAuthorityChange`, `OnRemoveComponent`, `OnCommandRequest` or `OnCommandResponse` method for the relevant schema type.
+
+##### Generated Code Format
+
+// describe type implementation here
+
+#### Manual (de)serialization
+
+You can also manually serialize and deserialize C API network operations. This is what generated code alternative described above does under the hood. To do this, you'll need to do the following:
+
+To send SpatialOS component updates and commands, use methods defined in the `SpatialWorkerConnection.h` file:
 
 * `void SendComponentUpdate(Worker_EntityId EntityId, const Worker_ComponentUpdate* ComponentUpdate);`
 * `Worker_RequestId SendCommandRequest(Worker_EntityId EntityId, const Worker_CommandRequest* Request, uint32_t CommandId);`
 * `void SendCommandResponse(Worker_RequestId RequestId, const Worker_CommandResponse* Response);`
 
-You access these methods via a reference to the net driver, as shown below:
+which can access via a reference to the net driver:
 
 `SpatialWorkerConnection connection = Cast<USpatialNetDriver>(World->GetNetDriver())->Connection;`
 
-There is a basic example in the _Examples_ section below. For more examples of how to construct component updates, command requests, and more, see the SpatialOS documentation on  [serialization in the Worker SDK in C’s API](https://docs.improbable.io/reference/latest/capi/serialization).
-
-#### Receive data
-
->**Note:** Your external SpatialOS components must have an ID between 1000 and 2000 to be registered by the pipeline.
-
-You set up your game to receive network operations using the `USpatialDispatcher::AddOpCallback` functions. These overloaded functions are parameterized with a `Worker_ComponentId` and a callback function const reference that takes one of the following network operation types as an argument:
+To receive [network operations](https://docs.improbable.io/reference/latest/shared/design/operations) for external SpatialOS components, you must provide custom callbacks for specific component IDs and operation types via the `USpatialDispatcher::AddOpCallback` functions. These overloaded functions are parameterized with a `Worker_ComponentId` and a callback function const reference that takes one of the following network operation types as an argument:
```

Comment:
I think `AddOpCallback` is out of date - https://github.com/spatialos/UnrealGDK/blob/master/SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialDispatcher.h

Reply:
yep nice, this should've been done in previous PR, make separate PR now